pkgs/by-name/in/indi-with-drivers/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  buildEnv,
+  makeBinaryWrapper,
+  indilib ? indilib,
+  pname ? "indi-with-drivers",
+  version ? indilib.version,
+  extraDrivers ? [ ],
+}:
+
+buildEnv {
+  name = "${pname}-${version}";
+
+  paths = [ indilib ] ++ extraDrivers;
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  postBuild =
+    if extraDrivers == [ ] then
```

Comment:
It makes sense to invert the condition and use `lib.optionalString` here.

Reply:
Indeed, that is nicer, thanks for the suggestion!